app/indices/account_index.rb

```diff
@@ -16,16 +16,12 @@
 
   has :provider_account_id
   has :tenant_id
+  has :state
 
   if System::Database.oracle?
-    # FIXME: creating a CRC32 in Oracle needs a custom function ...
```

Comment:
So this was done because of before Sphinx 2.2.3 it was impossible to filter on String
https://freelancing-gods.com/thinking-sphinx/v3/common_issues.html#string_filters

We are using 2.2.11+ so we can remove it